test/e2e/system_df_test.go

```diff
@@ -60,6 +60,27 @@ var _ = Describe("podman system df", func() {
 		Expect(volumes[2]).To(Equal("1"))
 	})
 
+	It("podman system df with format", func() {
+		SkipIfRemote()
+		session := podmanTest.Podman([]string{"create", ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"volume", "create", "data1"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"create", "-v", "data1:/data", "--name", "container2", "busybox"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"system", "df", "--format", `{{.Type}}\t{{.Total}}\t{{.Active}}\t{{.Size}}\t{{.Reclaimable}}\n`})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		// if userFmt is set, so there is no header, it will be 3
+		Expect(len(session.OutputToStringArray())).To(Equal(3))
```

Comment:
I know everyone is tired of hearing me say it, but this is not really a test. "Number of lines of output" is really pretty useless and just gives false confidence. Worse, there is no point to creating a volume or container here because none of that is checked. It just adds to test run time needlessly.

I would recommend scratching this and just relying on the system test. If there is a strong need to stick with e2e tests, please at least add actual validation of the output.

Reply:
Tend to agree, given this is just a duplication of the system test.

Reply:
all has be done